index.bs

```diff
@@ -7422,23 +8776,39 @@ partial dictionary MLOpSupportLimits {
 ### slice ### {#api-mlgraphbuilder-slice}
 Produce a slice of the input tensor.
 <script type=idl>
+dictionary MLSliceOptions : MLOperatorOptions {
+  sequence<[EnforceRange] unsigned long> strides;
+};
+
 partial interface MLGraphBuilder {
   MLOperand slice(MLOperand input,
                   sequence<[EnforceRange] unsigned long> starts,
                   sequence<[EnforceRange] unsigned long> sizes,
-                  optional MLOperatorOptions options = {});
+                  optional MLSliceOptions options = {});
 };
 
-partial  dictionary MLOpSupportLimits {
+partial dictionary MLOpSupportLimits {
   MLSingleInputSupportLimits slice;
 };
 </script>
+
+{{MLSliceOptions}} has the following members:
+<dl dfn-type=dict-member dfn-for=MLSliceOptions>
+    : <dfn>strides</dfn>
+    ::
+        The stride to step over each input along each axis.
+        The length of the strides array must equal the [=MLOperand/rank=] of the input tensor.
+        The the default is an array of length [=MLOperand/rank=] consisting of all 1's.
```

Comment:
Double "the"

Reply:
FYI, we have a lint rule to catch duplicate words, but (1) it was case sensitive and (2) it didn't handle words at the start or end of a line. I'll fix in an update.

Reply:
Deleted the the extra word.